src/Carousel.vue

```diff
@@ -548,8 +536,6 @@ export default {
         return;
       }
 
-      // we are good to prevent the move and handle the translation
-      e.preventDefault();
```

Comment:
when this line was written, maybe the author was still using chrome 56-, and option passive defaults to false.